tests/ccb-tests.js

```diff
@@ -8,20 +8,21 @@ function isNumeric(n) {
   return !isNaN(parseFloat(n)) && isFinite(n);
 }
 
-describe.skip('when executing ccb', function() {
+describe('when executing ccb', function() {
+  this.timeout(10000);
+
   var result;
+  // README: Add your jira username and password to this object before running tests.
   var jiraConfig = {
     protocol: 'https',
     host: 'opentable.atlassian.net',
     port: 443,
-    // FIXME: need test user. sent an email to techtable - max
     username: 'FIXME',
     password: 'FIXME',
     verbose: true,
     version: '2',
-    // FIXME: need test project. sent an email to techtable - max
-    project_id: '19102',
-    ccb_issue_type: '3',
+    project_id: '19200',
```

Comment:
Yah this is the test project that I mentioned in chat, I'll leave a comment with a link.